utils/releaser/src/ReleaseWorker/ReleaseCandidate/UpdateUpgradeReleaseWorker.php

```diff
@@ -83,7 +83,7 @@ public function getStage(): string
      */
     private function updateUpgradeFileWithReleasedVersion(Version $version, string $initialBranchName): void
     {
-        $upgradeFilePath = getcwd() . '/UPGRADE-' . $initialBranchName . '.md';
+        $upgradeFilePath = getcwd() . '/UPGRADE-' . $version->getMajor()->getValue() . '.' . $version->getMinor()->getValue() . '.md';
```

Comment:
This is effectively the same, or? 
I mean 14.0.1 is released from base branch 14.0, 15.2.0 from 15.2...